hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -1609,7 +1609,7 @@ You can also use an Access Point name as a path URI such as `s3a://finance-team-
 configuring the `.accesspoint.arn` property as a per-bucket override:
```

Comment:
i think this whole paragraph and example can be cut; as the only way to set an AP binding is on a per-bucket basis.

Reply:
I have removed the "you can also" section and example along.